app/code/core/Mage/Sales/Model/Observer.php

```diff
@@ -60,7 +60,7 @@ public function cleanExpiredQuotes($schedule)
 
             $quotes->addFieldToFilter('store_id', $storeId);
             $quotes->addFieldToFilter('updated_at', array('to'=>date("Y-m-d", time()-$lifetime)));
-            $quotes->addFieldToFilter('is_active', 0);
+            $quotes->addFieldToFilter('is_active', 1);
```

Comment:
Note: We can also remove the line

Reply:
`is_active` fitlering should be removed, otherwise, non-active quotes would not be deleted.